distributions/kfdef/kfctl_openshift_v1.3.0.yaml

```diff
@@ -80,5 +80,5 @@ spec:
     name: kubeflow-apps
   repos:
   - name: manifests
-    uri: https://github.com/kubeflow/manifests/archive/v1.3-branch.tar.gz
-  version: v1.3-branch
+    uri: https://github.com/kubeflow/manifests/archive/v1.3.0.tar.gz
+  version: v1.3.0-tag
```

Comment:
Is "-tag" (or -branch as it was before) the common way to reference things?  Not terribly important, I guess, but it's probably good to go along with whatever the standard is across the project.

Reply:
@crobby  so it was branch before because we were pulling from a branch and this broke our distribution because branch is not stable. We moved it to tag, and I placed tag in the version to indicate that. This```version``` is not used anywhere and I don't think we have standard for the format.